examples/fluentui-app/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "fluentui-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fluentui/react": "^7.155.3",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "@types/jest": "^26.0.15",
+    "@types/node": "^12.0.0",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.3",
+    "typescript": "^4.1.2",
+    "web-vitals": "^1.0.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-app-rewired build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "flamegrill": "./../../node_modules/.bin/flamegrill -t perf -o perf"
```

Comment:
The `node_modules` pathing really shouldn't be needed anywhere. Install location can vary depending on package manager and hoisting status, so using `node_modules` pathing at all is generally an antipattern.

I believe this should just work as `"flamegrill": "flamegrill -t perf -o perf"` since `flamegrill` is a dep. Although this raises an interesting question: why is `flamegrill` not a devDep?

Reply:
Ah, excellent point, thanks!